noxfile.py

```diff
@@ -31,18 +30,6 @@ def get_author_list():
     return sorted(authors, key=lambda x: x.lower())
 
 
-# -----------------------------------------------------------------------------
-# Ad-hoc commands
-# -----------------------------------------------------------------------------
-@nox.session
-def validate_news(session):
-    session.install("rstcheck")
-
-    news_files = sorted(glob.glob("news/*"))
-
-    session.run("rstcheck", *news_files)
```

Comment:
Which check covers this?

Reply:
`rst-backticks`

Reply:
It also found a couple of instances that `rstcheck` wasn't flagging: https://github.com/pypa/pip/pull/7078/commits/c47e1b6e5d508d60349718907b81664e14b4f514

Reply:
I think having `rst-backticks` is a good idea, but it is not a replacement. `rstcheck` is a more general rst syntax checker, and it wasn't flagging because single-backticks are valid rst.

Reply:
We could factor `rstcheck` into a pre-commit hook itself if there's not one already.

Reply:
I don't think we'd need it for anything else in the fragments, tbh.

Would you be okay if we add rstcheck as a hook, in a follow up PR? Or do you strongly prefer that I add it in this PR?

```yaml
-  repo: local
    hooks:
    -   id: rstcheck
        name: rstcheck
        entry: rstcheck
        files: '.rst'
        language: python
        additional_dependencies: [rstcheck, sphinx]
```

Reply:
A followup is OK with me (if at all). I agree, only the backticks are likely to be an issue in the news files.